src/terms/local_nonlinearity.jl

```diff
@@ -22,14 +23,14 @@ function ene_ops(term::TermLocalNonlinearity, basis::PlaneWaveBasis{T}, ψ, occu
 end
 
 
-function compute_kernel(term::TermLocalNonlinearity, ::AbstractBasis; ρ, kwargs...)
```

Comment:
`::AbstractBasis{T}`

Reply:
[080a1f7](https://github.com/JuliaMolSim/DFTK.jl/pull/779/commits/080a1f776820c71b32062ef47e7d00268f13547b)
and
[5f41cad](https://github.com/JuliaMolSim/DFTK.jl/pull/779/commits/5f41cad0d42f469334cb0dbbeacf90975e21b358)